openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -205,11 +205,40 @@ def get_managed_tool_data(self, tool):
 
         return data
 
+    def get_pre_create_attr_defs(self):
+        """Settings for create page"""
+        attr_defs = [
+            self._get_render_target_enum(),
+            self._get_reviewable_bool(),
+        ]
+        return attr_defs
+
     def get_instance_attr_defs(self):
-        return [
-            BoolDef(
-                "review",
-                default=True,
-                label="Review"
-            )
+        """Settings for publish page"""
+        attr_defs = [
+            self._get_render_target_enum(),
+            self._get_reviewable_bool(),
         ]
+        return attr_defs
+
+    # These functions below should be moved to another file
+    # so it can be used by other plugins. plugin.py ?
+
+    def _get_render_target_enum(self):
+        rendering_targets = {
+            "local": "Local machine rendering",
+            "frames": "Use existing frames",
+        }
+        if "farm_rendering" in self.instance_attributes:
+            rendering_targets["farm"] = "Farm rendering"
+
+        return EnumDef(
+            "render_target", items=rendering_targets, label="Render target"
+        )
+
+    def _get_reviewable_bool(self):
+        return BoolDef(
+            "review",
+            default=("reviewable" in self.instance_attributes),
+            label="Review",
+        )
```

Comment:
I know the way this is structured in code comes from Nuke, but I honestly don't really understand why - if this is just for a single creator - we need to change this into separate methods for reviewable bool and render target enum. I feel like this could just be done directly in `get_instance_attr_defs` instead or alike. Anyway, not a huge issue with it myself but like the comment described - it doesn't feel in the right place currently.

Reply:
Currently it's only 1 creator that uses it but I wish to also add a Rerender creator that also would need to access the `_get_render_target_enum` function and other creators in the future might also need that.

"Biggest" question is where to put that function.

Reply:
Rerender creator? How would it differ?

Reply:
Same question really - how does a pre-render creator differ? It's still a saver node, with start and end frames?
Just wondering how the behavior differs for a "prerender".